### PR TITLE
Add ansible playbooks for running tests remotely and nightly.

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: ansible/roles/kstest-nightly/templates/run_nightly_tests.j2 [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+### Sync results to remote host
+
+if [[ -n "${REMOTE_RESULTS_PATH}" ]]; then
+    rsync -avz ${RESULTS_PATH} ${REMOTE_RESULTS_PATH}
+    if [[ "${REMOTE_RESULTS_KEEP_LOCAL}" == "no" ]]; then
+        rm -rf ${RESULT_PATH}
+    fi
+fi
```
Comment (jkonecny12) [inline ansible/roles/kstest-nightly/templates/run_nightly_tests.j2]: Do we need this? It is another complex bash script and we want to get rid of them, thought.

File: ansible/roles/kstest/files/authorized_keys/kstests.pub [hunk 2]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJjCafb/FZ4hIuQXfUPvhkZmQT2na//fl2rmnG5i65EyJXrWZyO2VM4n4rDmYT2scxp3BDV1ovacTetiqUW3cBID+pzuyVmRonsE6eNudlP47BWv14kPY3SocImt4lgjA2bag4yhgN7mx20c3bLGL7L25D27SZwDYfBbPUl1uuypmMlyiLZ7Cwpf+qEeobfk5nwjZMDZEL9abPk2HYIm+2udLCqUtVj/51lQ+XL5AH1EJvMdh4Fv6GuqV69jSZoc9XcPoUX22NXIirym0Tpzro/j4xOZ0jDN5E012fdMcKwFwLMiXrSq1Amgk4B7z0EQQ4z3/by/EAsxCEkyuoa5vr Generated-by-Nov
```
Comment (jkonecny12) [inline ansible/roles/kstest/files/authorized_keys/kstests.pub]: Again, we don't want to have SSH key here, even when it is public.
Ask this question: What is the benefit for someone who checked out this repo and is outside of our team.

File: ansible/roles/kstest/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+---
+
+### Unlock Cloud Image root
+
+- name: Unlock root
+  user:
+    name: root
+    password: anaconda
```
Comment (jkonecny12) [inline ansible/roles/kstest/tasks/main.yml]: no, no, no, no...

File: ansible/roles/kstest/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+    validate: 'visudo -cf %s'
+
+- name: Add kstest to wheel group
+  user:
+    name: kstest
+    groups: wheel
+    append: yes
+
+- name: Allow authentization as root for Vritual Manager
```
Comment (jkonecny12) [inline ansible/roles/kstest/tasks/main.yml]: Typo Vritual -> Virtual

**Comment by rvykydal** [comment]
I opened a new PR:
https://github.com/rhinstaller/kickstart-tests/pull/136

**Pull request closed without merging**